src/ROUTES.js

```diff
@@ -49,6 +49,8 @@ export default {
     VALIDATE_LOGIN_WITH_VALIDATE_CODE: 'v/:accountID/:validateCode',
     ENABLE_PAYMENTS: 'enable-payments',
     WORKSPACE_NEW: 'workspace/new',
+    WORKSPACE_CARD: 'workspace/card',
+    BANK_ACCOUNT_NEW: 'bankAccount/new',
```

Comment:
Can we instead make this 

```js
getBankAccountRoute: (stepToOpen) => `bank-account/${stepToOpen}`
```

then we can use it like so `ROUTES.getBankAccountRoute('new')`

Reply:
Sure. I was going to ask about this `ADD_PERSONAL_BANK_ACCOUNT` route on the top. Does this exist for the same purpose?

Reply:
If you merge `main` you'll see some new changes and it should make more sense hopefully but the `bank-account` routes have been changed to:

https://github.com/Expensify/Expensify.cash/blob/ead7d7cc4c5467a9f75965411a1f12869cb65b05/src/ROUTES.js#L11-L12

and we handle the route a bit differently here by just taking the param 
https://github.com/Expensify/Expensify.cash/blob/ead7d7cc4c5467a9f75965411a1f12869cb65b05/src/pages/ReimbursementAccount/ReimbursementAccountPage.js#L83-L98